src/ts/components/seekbar.ts

```diff
@@ -207,46 +207,62 @@ export class SeekBar extends Component<SeekBarConfig> {
     let isUserSeeking = false;
     let isPlayerSeeking = false;
 
+    const getPlaybackPositionPercentage = (): number => {
```

Comment:
I guess it is a preference thing, 

but why do you use `const doStuff = () => ` instead of `private doStuff function() {..}`?

Reply:
Updated here https://github.com/bitmovin/bitmovin-player-ui/pull/414/commits/d0781dd012a4a1c057f7d1d476bd8d9a0164b893